nodejs/week1/warmup.js

```diff
@@ -0,0 +1,22 @@
+class Circle {
+  constructor(radius) {
+    this.radius = radius;
+  }
+  getDiameter() {
+    return 2 * this.radius;
+  }
+  getCircumference() {
+    return 2 * Math.PI * this.radius;
+  }
+  getArea() {
+    return Math.PI * this.radius * this.radius;
```

Comment:
You can also get `number` to the power of `x` as `Math.pow(number, x)` or `number**x`. 👍